BrowserKit/Sources/Redux/Action.swift

```diff
@@ -10,14 +10,22 @@ import Foundation
 /// action, including a UUID for a particular window. If an Action is truly global in
 /// nature or can't be associated with a UUID it can send either the UUID for the active
 /// window (see `WindowManager.activeWindow`) or if needed `WindowUUID.unavailable`.
```

Comment:
Just a note that this comment above needs to be updated if we go with this approach.